Gemfile

```diff
@@ -43,7 +52,7 @@ end
 
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
+  gem 'web-console', '~> 3.0'
```

Comment:
This is the sample project for Gentelella only.
It's better to just add necessary Gems